Error while loading shared libraries: libXss.so.1

Today we start to show the following error on RUM test

```
AssertionError: Expected done, got Failed to launch chrome! /rumjs-integration-test/node_modules/puppeteer/.local-chromium/linux-686378/chrome-linux/chrome: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory   TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md
```

--- a/docker/rum/Dockerfile
+++ b/docker/rum/Dockerfile
@@ -5,7 +5,7 @@ ARG RUM_AGENT_REPO=elastic/apm-agent-rum-js
 ARG APM_SERVER_URL
 
 RUN apt update -qq \
-    && apt install -qq -y curl git gnupg libgconf-2-4 --no-install-recommends \
+    && apt install -qq -y curl git gnupg libgconf-2-4 libxss1 --no-install-recommends \
     && curl -sSfkL https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
     && apt update -qq \

